packages/common/container-definitions/api-report/container-definitions.api.md

```diff
@@ -381,6 +381,14 @@ export interface IFluidPackageEnvironment {
 
 export { IGenericError }
 
+// @beta
```

Comment:
i think i would prefer alpha over beta, as we can bring it forward to beta if we have a need, but harder to push back to alpha